src/screens/Assets/collectibles/CollectiblesTab.js

```diff
@@ -140,7 +161,7 @@ const useSectionData = (numberOfColumns: number, expandItemsPerChain: FlagPerCha
 
   return chains.map((chain) => {
     const items = assetsPerChain[chain] ?? [];
-    const data = expandItemsPerChain[chain] ? chunk(items, numberOfColumns) : [];
+    const data = expandItemsPerChain[chain] ? Array(1).fill(chunk(items, numberOfColumns)) : [];
```

Comment:
I don't get it, it creates an array of all the items as single? seems redundant as `chunk` already returns the items? 🤔 also it can just be `[chunk(items, numberOfColumns)]`

Reply:
chunk is just to seperate each row of two elements each and if its multiple rows we will see multiple pagination components since pagination i have put is for a single row not multiple rows and also tried multiple rows but sections are there which seperates each chain so thats y putting it all in a single array and swiperflatlist will take care of all the slides with 2 nfts each